src/main/java/com/github/instagram4j/instagram4j/requests/users/UsersBlockedRequest.java

```diff
@@ -0,0 +1,18 @@
+package com.github.instagram4j.instagram4j.requests.users;
+
+import com.github.instagram4j.instagram4j.requests.IGGetRequest;
+import com.github.instagram4j.instagram4j.responses.users.UsersBlockedResponse;
+
+public class UsersBlockedRequest extends IGGetRequest<UsersBlockedResponse> {
```

Comment:
Hi sorry to nitpick but naming convention for request should follow similarly to the request path + Request appended at the end.

So this would be UsersBlockedListRequest (patrh is /users/blocked_list/